test/unit/runner.spec.js

```diff
@@ -473,37 +460,56 @@ describe('Runner', function() {
         throw new Error('this error will not propagate');
       });
       var runner = new Runner(suite);
-      try {
+      expect(function() {
         runner.hook('beforeEach', function() {});
-        expect(true, 'to be', true);
-      } catch (err) {
-        expect(false, 'to be', true);
-      }
+      }, 'not to throw');
     });
 
-    it('should allow unhandled errors in sync hooks to propagate through', function() {
+    it('should allow unhandled errors in sync hooks to propagate through', function(done) {
       suite.beforeEach(function() {
         throw new Error('allow unhandled errors in sync hooks');
       });
       var runner = new Runner(suite);
       runner.allowUncaught = true;
-      function throwError() {
-        runner.hook('beforeEach', function() {});
-      }
-      expect(throwError, 'to throw', 'allow unhandled errors in sync hooks');
+
+      var EVENT_HOOK_BEGIN = 'hook';
+      runner.on(EVENT_HOOK_BEGIN, function(hook) {
+        var _run = hook.run;
+        hook.run = function() {
+          function throwError() {
+            _run.call(hook);
+          }
+          var expected = 'allow unhandled errors in async hooks';
```

Comment:
should be "sync", not "async": `var expected = 'allow unhandled errors in sync hooks';`

Reply:
ah! probably a typo introduced during conflicts resolution, thanks